framework/include/auxkernels/AuxKernel.h

```diff
@@ -11,7 +11,7 @@
 #define AUXKERNEL_H
 
 #include "MooseObject.h"
-#include "MooseVariableField.h" // for name() in mooseError calls
+#include "MooseVariableFEImpl.h" // for name() in mooseError calls
```

Comment:
This comment lies, `name()` comes from `MooseObject`.

Reply:
Definitely delete, because there are 2 kinds of `name()`:

1. `name()` from `MooseObject` that this `AuxKernel` gets (the one I was thinking about)
2. `name()` from `MooseVariableBase` (the one you mantioned)

So, the comment is (a) confusing, (b) not needed :-)